code/modules/vehicles/atv.dm

```diff
@@ -0,0 +1,81 @@
+
+/obj/vehicle/atv
+	name = "All-terrain vehicle"
```

Comment:
took from the original (granted I moved it's ```callme``` to it's name as I deprecated that var, but yeah I'll sort it)